mvc-login/build.gradle

```diff
@@ -6,23 +6,22 @@ buildscript {
 
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '2.7.0'
+    id 'org.springframework.boot' version '3.1.4'
     id 'io.spring.dependency-management' version '1.1.0'
 }
 
 group = 'com.auth0'
 version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '1.8'
```

Comment:
A brand new app has:

```groovy
java {
    sourceCompatibility = '17'
}
```

Reply:
👍 incorporated in commit along with other changes